packages/core/src/lib/map/map.ts

```diff
@@ -433,3 +458,15 @@ export function mapArray<
 
   return destinationArray;
 }
+
+function setMemberMutateFn(destinationObj: Record<string, unknown>, destinationMember: string[]) {
```

Comment:
question: is there a reason why you don't want to do curry here?

Reply:
missed this, thank for response.